src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  box-sizing: border-box;
+  height: $card-height;
+  width: $card-width;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    height: $image-height;
+    width: $image-width;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    margin-top: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    margin-top: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
```

Comment:
The variable `$secondary-color` is used for the `color` property of the `&__code` class. Ensure that this variable is defined elsewhere in your SCSS files. If it's not defined, you need to declare it with a valid color value before using it.